client/rest/src/test/java/org/elasticsearch/client/documentation/RestClientDocumentation.java

```diff
@@ -327,6 +327,15 @@ public void commonConfiguration() throws Exception {
                     });
             //end::rest-client-config-timeouts
         }
+        {
+            //tag::rest-client-config-request-options-timeouts
+            RequestOptions.Builder builder = RequestOptions.DEFAULT.toBuilder();
+            RequestConfig.Builder requestConfigBuilder = RequestConfig.custom();
+            requestConfigBuilder.setConnectTimeout(5000).setSocketTimeout(60000);
+            builder.setRequestConfig(requestConfigBuilder.build());
+            RequestOptions options = builder.build();
```

Comment:
This code is difficult to make sense of quickly; A lot of repeated words like "Request" and "Builder" in the snippet makes it seem repetitive and a bit hard to find the API boundary between the two config classes. Perhaps it could be reworked to make it easier to read using the fluent methods?

```
RequestConfig requestConfig = RequestConfig.custom()
    .setConnectTimeout(5000)
    .setSocketTimeout(60000)
    .build();
RequestOptions options = RequestOptions.DEFAULT.toBuilder()
    .setRequestConfig(requestConfig)
    .build();
```

Reply:
yeah, it's a good idea. thank you!